config/routes.rb

```diff
@@ -1,4 +1,14 @@
 Rails.application.routes.draw do
-  root to: 'pages#home'
+  devise_for :users
+  root to: 'gardens#home'
```

Comment:
@georgkloeck - did you see the pages controller they had us start with?  That's what we decided to use for the root.

Reply:
But we dont have a pages controller?
so the root should be on the gardens controller?

Reply:
@georgkloeck - yes, we do.  So we'll use that for root